doc/odoc.mld

```diff
@@ -23,7 +23,7 @@ val float_dsig : int -> float t
     and floats. *)
 ]}
 
-These comments are picked up by [odoc] and {{!Fmt.float_dsig}turned into html}, or TeX or manpages.
+These comments are picked up by [odoc] and {{!Fmt.float_dsig}turned into HTML}, TeX, or manpages.
```

Comment:
Should this be TeX or LaTeX?

Reply:
LaTeX !

Reply:
Glad I asked! I thought TeX was a little odd.